.gitlab-ci.yml

```diff
@@ -121,3 +122,16 @@ integrate:
     - !reference [.update_pin, needs]
   rules:
     - !reference [.update_pin, rules]
+
+# used by release
+.python_versions:
+  parallel:
+    matrix:
+      - BUILD_PY_VERSION: [python3.6]
+      - RELEASE_PY_VERSION: [python3.6]
+
+release:
+  before_script:
+    - git fetch --unshallow
+  variables:
+    LEGACY_BASE_CONTAINER_VERSION: 2.3.0
```

Comment:
Can we add this to the gitlab notes, it should cover expectations like you need to have a publish step defined in `tox.ini`

Reply:
Added, thank you:
https://github.com/NCI-GDC/gdc-dev-guide/wiki/Gitlab-Notes#release-to-nexus